.github/workflows/ci.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 env:
   python-version: "3.9"
-  rust-version: 1.65.0
```

Comment:
This line can be entirely removed given the action `actions-rust-lang/setup-rust-toolchain` directly uses `rust-toolchain.toml`